src/generator/gentime/settingsSingleton.ts

```diff
@@ -7,17 +7,26 @@ export namespace Gentime {
      */
     projectIdIntToGraphQL?: 'ID' | 'Int'
     /**
-     * TODO
+     * Should Prisma Schema docs propagate as docs?
+     *
+     * @default `true`
```

Comment:
```suggestion
     * @default true
```

JSDoc doesn't accept backticks here.

Reply:
It would be nice to show some examples of the effect when it is off vs on.

JSDoc `@example` only allows JS source code so might need some exploration on how best to do this. For example instead of using `@example` tag using markdown code blocks.

Reply:
I tried to use markdown code blocks but `@homer0/prettier-plugin-jsdoc` breaks the formatting of code.

Reply:
Ok, please just leave that as a `// TODO ...` comment for now 🙏 

This PR still brings incremental value. Let's not block just b/c its not 100%.